master/internal/api_notebook.go

```diff
@@ -53,11 +53,11 @@ const (
 	notebookDefaultPage = "/run/determined/workdir/README.ipynb"
 )
 
-var notebooksAddr = actor.Addr(command.NotebookActorPath)
+var notebooksAddr = actor.Addr("tensorboard")
```

Comment:
should this be "notebooks"? it might be good to keep those constants around

Reply:
rather, it looks like there is an ask or actor in this that might need to be updated to remove notebooksAddr

Reply:
oh this is just vestigial code that I edited to make the code compile partway through -- it's not necessary now, so I'll delete that line